### PR TITLE
Emulate recent file picker (not including USS files)

### PR DESCRIPTION
**Pull request by katelynienaber**
Fresh PR for this enhancement.

Signed-off-by: katelynienaber <katelyn.nienaber@broadcom.com>

### CODE REVIEW

**Comment by Colin-Stone** [comment]
Now this has merged with master you will need to add the API's to the interface. Will the same features be available for uss?.

openRecent/recentMemberPrompt doesn't really convey what the feature is doing

**Comment by katelynienaber** [comment]
> Now this has merged with master you will need to add the API's to the interface. Will the same features be available for uss?.
> 
> openRecent/recentMemberPrompt doesn't really convey what the feature is doing

Okay. I am working on USS today. I was going to make a separate pr for USS.

Will change names, give me 5 mins

File: src/api/IZoweTree.ts [hunk 1]
```diff
@@ -180,6 +180,34 @@ export interface IZoweTree<T> extends vscode.TreeDataProvider<T> {
      * Renames a node based on the profile and it's label
      * @deprecated should not be visible outside of class
      *
```
Comment (Colin-Stone) [inline src/api/IZoweTree.ts]: The sequence of methods is broken. The method renameNode is disconnected from it's definition

Comment (katelynienaber) [reply src/api/IZoweTree.ts]: Fixed ⚡️

**Comment by katelynienaber** [comment]
> Thanks Katelyn this is a really nice function

Will probably change it a bit to add USS as well but thanks :)

**Comment by katelynienaber** [comment]
Closing because opened PR #627 to include USS items.

**Pull request closed without merging**